fleetshard/main.go

```diff
@@ -62,6 +66,123 @@ func main() {
 		}
 	}()
 
+	glog.Info("Creating certMonitor")
+
+	certmonitorConfig := &certmonitor.Config{
+
+		Monitors: []certmonitor.MonitorConfig{
+			{
+				Namespace: certmonitor.SelectorConfig{
+					LabelSelector: &metav1.LabelSelector{
+						MatchExpressions: []metav1.LabelSelectorRequirement{
+							{
+								Key:      "rhacs.redhat.com/tenant",
```

Comment:
Could you use the constant instead of hardcoded string? (from fleetshard/pkg/central/reconciler/reconciler.go)
```suggestion
								Key:      reconciler.TenantIDLabelKey,
```